File: sprint5/By languages/Java/lesson4code1.java
void insert_node(Node root, int key) {
  if (key < root.key) {
    if (root.left == null) {
      root.left = new Node(key);
    } else {
      insert_node(root.left, key);
    }
  } else {
    if (root.right == null) {
      root.right = new Node(key);
    } else {
      insert_node(root.right, key);
    }
  }
}

File: sprint5/By languages/python/lesson4code1.py
def insert_node(root, key):
  if key < root.key: 
	  if root.left is None:
      root.left = Node(key)
    else:
      insert_node(root.left, key)
  if key >= root.key:
	  if root.right is None:
      root.right = Node(key)
    else:
      insert_node(root.right, key) 

